### ISSUE TITLE
Consider using viewBox instead of explicit width / height

### ISSUE THREAD

**Issue by wbobeirne** [urbit/sigil-js#30]
When specifying a size, it would be ideal if the svg element got a `viewBox="0 0 ${size} ${size}"` instead of fixed width and height. This would allow resizing it arbitrarily in CSS.

**Comment by shrugs** [comment]
very much +1

within the example ReactSVGComponents, I currently use:

```jsx
  svg: (p, key) => {
    const { width, height, ...attrs } = p.attr;
    return (
      <svg
        {...attrs}
        style={{ width: '100%', height: '100%' }}
        version={'1.1'}
        xmlns={'http://www.w3.org/2000/svg'}
        key={key}
        viewBox={`0 0 ${width} ${height}`}
        preserveAspectRatio="xMidYMid meet">
        {map(get(p, 'children', []), (child, idx) =>
          ReactSVGComponents[child.tag](child, idx)
        )}
      </svg>
    );
  },
```

to get a resizable svg

**Comment by g-a-v-i-n** [comment]
resolved in #39 and published in 1.3.5
